src/components/Tipseen/__stories__/tipseen.stories.js

```diff
@@ -94,11 +94,14 @@ export const Sandbox = () => (
       animationTypes={select("Animation types", Tipseen.animationTypes, Tipseen.animationTypes.EXPAND)}
       justify={select("justify", Tipseen.justifyTypes, Tipseen.justifyTypes.CENTER)}
       isCloseButtonHidden={boolean("Is close button hidden", false)}
+      hideWhenReferenceHidden={boolean("Hide when reference hidden", false)}
       content={
         <TipseenContent
           title={text("Title", "Title")}
-          isDismissHidden={boolean("Is dismiss hidden", false)}
+          isDismissHidden={boolean("Is dismiss hidden", true)}
           children={text("Tipseen content text", "Popover message will appear here loremipsum dolor samet…")}
+          submitButtonText={text("Submit button text", "Submit button text")}
```

Comment:
In evginy design the button called "submit" but in the monolith it's standard name is got it.
so, here the api remain the same but in the monolith I am using tipseen content wrapper which expect the receive "gotItButtonText" and "gotItCallback" (which there I also close the tipseen with the redux monolith mechanism).
I think that "gotItButtonText" and "gotItCallback" will be little bit weird inside an open source library api. what do you think?